disjoint-sets.lisp

```diff
@@ -58,28 +46,40 @@ Example:
 (disjoint-sets-find sets 5)
 "
   (let ((parent (aref sets id)))
-    (if (= id parent)
-        ;; If "id" is the root, just return it.
-        id
+    (if (= id parent) id             ; If `id' is the root, return it.
```

Comment:
I put two `;` when the comment applies to the line below and only one `;` when it applies to the same line.

So put back that semicolon where you fund it